src/ncp/ncp_base.hpp

```diff
@@ -575,10 +579,12 @@ class NcpBase
 #endif
     ThreadError InsertPropertyHandler_IPV6_ADDRESS_TABLE(uint8_t header, spinel_prop_key_t key,
                                                          const uint8_t *value_ptr, uint16_t value_len);
+#if OPENTHREAD_FTD
     ThreadError InsertPropertyHandler_THREAD_LOCAL_ROUTES(uint8_t header, spinel_prop_key_t key,
```

Comment:
Yes, I also agree that MTD could act as a border router, while in current implementation, MTD does't support border router functionality, please refer to [network_data_local_mtd.hpp](https://github.com/openthread/openthread/blob/master/src/core/thread/network_data_local_mtd.hpp#L52).

I removed border router related changes from this PR, and I will introduce `OPENTHREAD_ENABLE_BORDER_ROUTER` flag and address the issue in a separate PR, thanks.